### ISSUE TITLE
Are we allowed to add bookmarks in our chrome to direct yaml in the Tasks section of kubernetes.io?

### ISSUE THREAD

**Issue by nicoclau** [dgkanatsios/CKAD-exercises#159]
In Tasks we are lot of yamls direct links for example : https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/pods/resource/memory-request-limit.yaml

are we allowed to bookmark this for the exam?

Thanks

**Comment by shubhamrajvanshi** [comment]
Nope, you are only allowed to open one additional tab in order to access assets at https://kubernetes.io/docs/, https://github.com/kubernetes/,  https://kubernetes.io/blog/, and their subdomains. This includes all available language translations of these pages (e.g. https://kubernetes.io/zh/docs/)
No other tabs may be opened and no other sites may be navigated to   (including https://discuss.kubernetes.io/). 
The allowed sites above may contain links that point to external sites. It is the responsibility of the candidate not to click any links to navigate to a domain that is not allowed

**Comment by MaxAtoms** [comment]
This issue is now obsolete and can be closed. As of June 2022, personal browser bookmarks are not allowed anymore. Instead, the Firefox browser within the exam environment must be used.

Sources:
https://docs.linuxfoundation.org/tc-docs/certification/certification-resources-allowed#certified-kubernetes-administrator-cka-and-certified-kubernetes-application-developer-ckad

https://training.linuxfoundation.org/bridge-migration-2021/
